### ISSUE TITLE
Change blue to cyan for skipped tests

### ISSUE THREAD

**Issue by dougthor42** [CleanCut/green#59]
The blue color tends to not show up very well under windows CMD:

![image](https://cloud.githubusercontent.com/assets/5386897/7966954/116c6406-09dc-11e5-91dd-5d6465dd80af.png)
![image](https://cloud.githubusercontent.com/assets/5386897/7967163/5452c890-09dd-11e5-9a5c-e521c5dc4be2.png)

If changed to cyan:
![image](https://cloud.githubusercontent.com/assets/5386897/7967082/dca695f6-09dc-11e5-9f9b-4f9649d13ee8.png)
![image](https://cloud.githubusercontent.com/assets/5386897/7967178/5c82b854-09dd-11e5-87f0-7a6e25954624.png)

Diff for output.py
```
--78: return termstyle.blue(text)
++78: return termstyle.cyan(text)
```

**Comment by CleanCut** [comment]
Sorry for the slow response, I've been swamped in my day job lately.

Question: Is that the default "blue" color on Windows?  (if so, what a stupid default color to choose for blue against black, microsoft)

In my experience with Linux and OS X, the default color scheme has blue a readable color, and if people customize their color scheme, then they ought to make sure blue ends up readable.

But if the default is unreadable, then I will probably change it to cyan just for Windows.  (Seems like all the special cases are "just for Windows").

**Comment by dougthor42** [comment]
Haha yeah, it's the default color for Windows CMD :-(

Thanks,

**Comment by CleanCut** [comment]
I fixed it in the upcoming 1.9.3 release.  Would you like me to mention you in the changelog?  If so, would you prefer I use your user name or real name?

**Comment by dougthor42** [comment]
No need to mention me. :-)  Though if you do, I guess I'd prefer username.

Thanks!
